File: Trees/binary-tree.py
""" Introduction to Binary Tree """

class Node:
    def __init__(self, data):
        self.right=None
        self.left=None
        self.key=data



root=Node(1)
root.left=Node(2)
root.right=Node(3)
root.left.left=Node(4)

File: Java/Trees/BinaryTree.java
package tree;

import tree.Node;

public class BinaryTree {

	Node root;
	
	BinaryTree(int key){
		root=new Node(key);
	}
	
	BinaryTree(){
		root=null;
	}
	
	
	public static void main(String args[]){
		BinaryTree tree=new BinaryTree();
		
		tree.root=new Node(1);
		tree.root.left=new Node(2);
		tree.root.right=new Node(3);
		
		
	}
}
